BO4E/ENUM/BoTyp.cs

```diff
@@ -18,6 +18,9 @@ public enum BoTyp
         /// <summary>1</summary>
         GESCHAEFTSPARTNER,
 
+        /// <summary>1</summary>
+        HANDELSUNSTIMMIGKEIT,
```

Comment:
Es kann gefährlich sein ein Enum-Wert in der Mitte einzufügen. Weil es intern als Zahl interpretiert wird. Wenn zwei Services miteinander Kommunzieren, die unter schiedliche Version haben, dann könnte z.B. eine Marktlokation versendet werden aber eine Handelsunstimmigkeit verstanden werden.

Reply:
oh weia, ok, dann pack ichs lieber unten hin

Reply:
3c2f203b333d01281b091f09c5bec67a07eee607